README.md

```diff
@@ -30,12 +30,22 @@
 By automating the early stage of the machine learning pipeline, our end user can easily define a task and solve it. See the [documentation](https://compose.alteryx.com) for more information.
 
 ## Install
-Compose can be installed for Python 3.6 or later by running the following command:
+Compose is available on [PyPI](https://pypi.org/project/composeml/) and [Conda-forge](https://anaconda.org/conda-forge/composeml) for Python 3.6 or later.
```

Comment:
let's drop these hyperlinks and rely on the ones in the specific install instructions